lib/ossert/stats/base.rb

```diff
@@ -89,6 +90,15 @@ def median(values, default_value: 0)
         (values[middle_idx - 1] + values[middle_idx]) / 2
       end
 
+      def get_count(metric)
```

Comment:
I will add comprehensive docs in the second phase, because there will be much more to describe. Hope it's ok.